WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -12,6 +12,9 @@
 import org.wordpress.android.widgets.WPTextView;
 
 public class DialogUtils {
+
+    private static OpenSansEditText editText;
```

Comment:
I noticed one more problem. If we select ok on the dialog box and then rotates the screen the dialog will open up because savedInstanceState will not be null. Can we use DialogFragment?